specification/resources/resource-manager/Microsoft.Solutions/stable/2019-07-01/managedapplications.json

```diff
@@ -1316,11 +1324,70 @@
           "readOnly": true,
           "$ref": "#/definitions/ProvisioningState",
           "description": "The managed application provisioning state."
+        },
+        "billingDetails": {
+          "readOnly": true,
+          "$ref": "#/definitions/ApplicationBillingDetailsDefinition",
+          "description": "The managed application billing details."
+        },
+        "additionalManagedScopes": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The additional managed scopes outside of the normal managed resource group. The key is the resource that manages the scope and the value is the fully qualified scope."
```

Comment:
Why represent this as a dictionary instead of an array of objects with this info?  A dictionary is less user friendly (you have to know what the key and value are supposed to be) and it means the properties will be dynamic and not addressable via common tooling like Azure Policy

Reply:
I did some research, this property is actually not exposed to end user through api. Will remove in next iteration.